internal/command/command.go

```diff
@@ -114,7 +158,19 @@ func (c Client) SearchForFlags(flags []string, ctxLines int, delimiters []rune)
 		return nil, errors.New(string(out))
 	}
 
-	grepRegexWithFilteredPath, err := regexp.Compile("(?:" + regexp.QuoteMeta(c.Workspace) + "/)" + grepRegex.String())
+	/*
+	   searchRegex splits resulting grep lines into groups
+	   Group 1: File path.
+	   Group 2: Line number.
+	   Group 3: Separator. A colon indicates a match, a hyphen indicates a context lines
+	   Group 4: Line contents.
+
+	   Example output from line search tool: /path/to/file/flags.go:2:ldClient.BoolVariation("my-flag-key", user, false)
+	   Example result: ["/path/to/file/flags.go:2:ldClient.BoolVariation(\"my-flag-key\", user, false)", "/path/to/file/flags.go", 2, ":", "ldClient.BoolVariation(\"my-flag-key\", user, false)"]
+	*/
+	searchRegex := regexp.MustCompile("(.*)[:-]([0-9]+)(:|-)(.*)")
+
+	searchRegexWithFilteredPath, err := regexp.Compile("(?:" + regexp.QuoteMeta(c.Workspace) + "[\\/]?)" + searchRegex.String())
```

Comment:
This regex wrapper has been changed slightly to support the windows path separator (backslash)